packages/components/checkbox/index.js

```diff
@@ -33,30 +39,30 @@ export const Checkbox = ({
   const Override = {
     wrapper: {
       alignItems: 'center',
-      lineHeight: 'checkbox',
+      lineHeight: wrapper.lineHeight,
       display: 'inline-flex',
       flexDirection: isReversed ? 'row-reverse' : 'row',
       justifyContent: isReversed ? 'flex-end' : 'start',
       cursor: isDisabled ? 'not-allowed' : 'pointer',
       ...getAllowedProps(override.wrapper, [...layout, ...typography, ...flex]),
     },
     controlBox: {
-      size: 'checkbox',
-      borderWidth: 'checkbox',
-      borderColor: 'borderColor.checkbox.default',
-      borderStyle: 'solid',
-      rounded: 'md',
-      _checked: {
-        bg: 'bg.checkbox.checked',
-        color: 'white',
-        borderColor: 'borderColor.checkbox.checked',
-      },
-      _disabled: { borderColor: 'borderColor.checkbox.disabled' },
-      _focus: { borderColor: 'outline' },
+      size: controlBox.normal.size,
+      borderWidth: controlBox.normal.borderWidth,
+      borderColor: controlBox.normal.borderColor,
+      borderStyle: controlBox.normal.borderStyle,
+      rounded: controlBox.normal.rounded,
```

Comment:
Name is perfect so that it can be spread. Let them see the config file to adjust the overrides.
```suggestion
      [...controlBox.normal]
```

Reply:
Yes, this can be done like that as well.
I will change that today when moving the styles out of the component.

Reply:
This is done